README.md

```diff
@@ -3,42 +3,47 @@ About nose
 
 Home: http://readthedocs.org/docs/nose/
 
-Package license: LGPL-2.0
+Package license: LGPL-2.1
 
 Feedstock license: BSD 3-Clause
 
 Summary: Nose extends unittest to make testing easier
 
+NOTE: Nose has been in maintenance mode for the past several years and will
+likely cease without a new person/team to take over maintainership. New
+projects should consider using Nose2, py.test, or just plain
+unittest/unittest2.
```

Comment:
Description wasn't working before, but now is. Hence it is included even though the recipe is unchanged in this regard.